components/service/glean/src/main/java/mozilla/components/service/glean/net/HttpPingUploader.kt

```diff
@@ -67,6 +67,10 @@ internal class HttpPingUploader : PingUploader {
 
         val request = buildRequest(path, data, config)
 
+        config.pingTag?.let {
```

Comment:
This seems to belong more to `buildRequest`. We should also add a test case for this in `HttpPingUploaderTest.kt` and a comment in the code explaining what's that for.